bin/edit.js

```diff
@@ -26,6 +35,10 @@ function handleEdit(args) {
     var wtName  = args.name ? args.name + '/' : '';
     var url     = profile.url + '/edit/' + profile.container + '#/' + wtName + profile.token;
 
+    if (args.previousVersion) {
+        url = profile.url + '/prev/edit/' + profile.container + '#/' + wtName + profile.token;
```

Comment:
Would it make sense to make it a query param which value is the semver of the editor to use? E.g. `?version=1` or `?version=1.2.3`. The `previous-version` option of the CLI would also allow a semver to be specified.

This is primarily for giving ourselves a chance to provide people with a sensible workaround in case they run into trouble. We would not want the behavior behind `prev` to change over time I think.